File: occupy/Assets/Scripts/Network/SocketMessage.cs
using UnityEngine;
using System.Collections.Generic;
using UnityEngine.Scripting;
using System;

[Serializable]
public class SocketMessage {
	public string Id = System.Guid.NewGuid().ToString();
	public string PlayerKey = "";
	public string Cmd = "";
	public List<string> Params = new List<string>();
	public string ExceptionMessage="";

}
	

File: occupy-server/src/main/java/com/mcm/network/messages/SocketMessage.java
package com.mcm.network.messages;

import com.mcm.entities.mongo.gameObjects.playerObjects.Tower;

import java.util.LinkedList;
import java.util.List;
import java.util.UUID;

/**
 * Created by alirezaghias on 11/5/2016 AD.
 */
public class SocketMessage {
    public String Id = UUID.randomUUID().toString();
    public String Cmd = "";
    public LinkedList<String> Params = new LinkedList<>();
    public String PlayerKey = "";
    public String ExceptionMessage = "";

    @Override
    public String toString() {
        return "SocketMessage{" +
                "Id='" + Id + '\'' +
                ", Cmd='" + Cmd + '\'' +
                ", PlayerKey='" + PlayerKey + '\'' +
                ", ExceptionMessage='" + ExceptionMessage + '\'' +
                ", Params=" + Params +
                '}';
    }
}
